classes/CartRule.php

```diff
@@ -699,202 +704,203 @@ public function checkValidity(Context $context, $alreadyInCart = false, $display
                 if ($otherCartRule['id_cart_rule'] == $this->id && !$alreadyInCart) {
-                    return (!$display_error) ? false : Tools::displayError('This voucher is already in your cart');
+                    return (!$displayError) ? false : Tools::displayError('This voucher is already in your cart');
                 }
                 if ($otherCartRule['gift_product']) {
-                    --$nb_products;
+                    --$nbProducts;
                 }
 
                 if ($this->cart_rule_restriction && $otherCartRule['cart_rule_restriction'] && $otherCartRule['id_cart_rule'] != $this->id) {
                     $combinable = Db::getInstance()->getValue('
 					SELECT id_cart_rule_1
 					FROM '._DB_PREFIX_.'cart_rule_combination
-					WHERE (id_cart_rule_1 = '.(int)$this->id.' AND id_cart_rule_2 = '.(int)$otherCartRule['id_cart_rule'].')
-					OR (id_cart_rule_2 = '.(int)$this->id.' AND id_cart_rule_1 = '.(int)$otherCartRule['id_cart_rule'].')');
+					WHERE (id_cart_rule_1 = '.(int) $this->id.' AND id_cart_rule_2 = '.(int) $otherCartRule['id_cart_rule'].')
+					OR (id_cart_rule_2 = '.(int) $this->id.' AND id_cart_rule_1 = '.(int) $otherCartRule['id_cart_rule'].')');
                     if (!$combinable) {
-                        $cart_rule = new CartRule((int)$otherCartRule['id_cart_rule'], $context->cart->id_lang);
+                        $cartRule = new CartRule((int) $otherCartRule['id_cart_rule'], $context->cart->id_lang);
                         // The cart rules are not combinable and the cart rule currently in the cart has priority over the one tested
-                        if ($cart_rule->priority <= $this->priority) {
-                            return (!$display_error) ? false : Tools::displayError('This voucher is not combinable with an other voucher already in your cart:').' '.$cart_rule->name;
+                        if ($cartRule->priority <= $this->priority) {
+                            return (!$displayError) ? false : Tools::displayError('This voucher is not combinable with an other voucher already in your cart:').' '.$cartRule->name;
                         }
                         // But if the cart rule that is tested has priority over the one in the cart, we remove the one in the cart and keep this new one
                         else {
-                            $context->cart->removeCartRule($cart_rule->id);
+                            $context->cart->removeCartRule($cartRule->id);
                         }
                     }
                 }
             }
         }
 
-        if (!$nb_products) {
-            return (!$display_error) ? false : Tools::displayError('Cart is empty');
+        if (!$nbProducts) {
+            return (!$displayError) ? false : Tools::displayError('Cart is empty');
         }
 
-        if (!$display_error) {
+        if (!$displayError) {
             return true;
         }
     }
 
```

Comment:
Do you use PhpStorm?

Here are some settings to make it comply with Symfony standards:
![phpdocsettings](https://cloud.githubusercontent.com/assets/6775736/18357199/4a7eccd0-75f0-11e6-9622-21d27a3e74a3.png)